arches/app/templates/views/components/search/advanced-search.htm

```diff
@@ -82,18 +82,18 @@ <h4 class="search-facet-item-heading">
                         type="text" 
                         class="form-control"
                         data-bind="
-                            attr:{placeholder: $root.translations.find + '...'},
+                            attr:{placeholder: $root.translations.find + '...', 'aria-label': $root.translations.find + '...'},
                             value: facetFilterText, 
                             valueUpdate: 'keyup'
                         "
                     >
                     <!-- Clear Filter -->
-                    <span class="clear-node-search" data-bind="click: function () { facetFilterText(''); }"><i class="fa fa-times-circle"></i></span>
+                    <span tabindex="0" class="clear-node-search" data-bind="onEnterkeyClick, onSpaceClick, click: function () { facetFilterText(''); }"><i class="fa fa-times-circle"></i></span>
```

Comment:
I think we should add a aria-lable here.  Maybe `aria-label="{% trans "Clear facet filter" %}"`

Reply:
Added aria-label